transformers/events/new_cdp/transformer.go

```diff
@@ -71,8 +81,8 @@ func (t Transformer) ToModels(abi string, logs []core.EventLog, _ *postgres.DB)
 			ColumnValues: event.ColumnValues{
 				event.HeaderFK:      newCdpEntity.HeaderID,
 				event.LogFK:         newCdpEntity.LogID,
-				constants.UsrColumn: newCdpEntity.Usr.Hex(),
-				constants.OwnColumn: newCdpEntity.Own.Hex(),
+				constants.UsrColumn: usrAddressID,
+				constants.OwnColumn: ownAddressID,
```

Comment:
Wondering if we should change the `usr` and `own` fields in the database to be numerics and foreign keys to the address table.

Reply:
➕